pan3d/ui/pan3d_components/module/__init__.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+# Compute local path to serve
+serve_path = str(Path(__file__).parent.with_name("serve").resolve())
```

Comment:
serve should be nested into the module. Otherwise it is not a module if it needs files outside of its tree...

Reply:
I copied this code from https://github.com/Kitware/trame-components/blob/master/trame_components/module/__init__.py, so I'm not clear on what its intent is. Could you clarify what change you mean?

Reply:
Should be with the following path `pan3d/ui/module/`.

Reply:
No you did not, you've inserted `.parent`